Makefile

```diff
@@ -1,5 +1,5 @@
 RELEASE_REPO ?= public.ecr.aws/b6u6q9h4
-RELEASE_VERSION ?= $(shell git describe --tags --always --dirty)
+RELEASE_VERSION ?= $(shell git describe --tags --always)
```

Comment:
why'd you remove the `--dirty`?  I think that's an important flag to have since it indicates that there are changes that have not been committed.  If you start from a clean point w/ no changes when generating this, it shouldn't add the `-dirty` to the end of the version.

Reply:
I see why it was removed now. Let's chat about moving to a release process where the manifests are uploaded to github as an asset w/ gh actions rather than generated and committed.